pkgs/dart_model/benchmark/main.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+
+import 'buffer_maps_buffer_wire_benchmark.dart';
+import 'buffer_maps_json_wire_benchmark.dart';
+import 'builder_maps_builder_wire_benchmark.dart';
+import 'builder_maps_json_wire_benchmark.dart';
+import 'sdk_maps_buffer_wire_benchmark.dart';
+import 'sdk_maps_builder_wire_benchmark.dart';
+import 'sdk_maps_json_wire_benchmark.dart';
+import 'serialization_benchmark.dart';
+
+void main() {
+  final serializationBenchmarks = [
```

Comment:
Let's use a record for these so that below you don't have to rely on specific indexes